<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>2000000</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #2c3e50; padding: 10px;</string>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #2c3e50; padding: 20px;</string>
          </property>
          <property name="text">
           <string>PHARMACY MANAGEMENT SYSTEM</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordInput">
          <property name="styleSheet">
           <string notr="true">padding: 10px; border: 2px solid #3498db; border-radius: 5px;</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="adminLoginBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: #3498db;
│           color: white;
│           padding: 12px;
│           border-radius: 5px;
│           font-size: 14px;
│           font-weight: bold;</string>
          </property>
          <property name="text">
           <string>ADMIN LOGIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="staffLoginBtn">
          <property name="styleSheet">
           <string notr="true">background-color: #2ecc71;
  color: white;
  padding: 12px;
  border-radius: 5px;
  font-size: 14px;
  font-weight: bold;</string>
          </property>
          <property name="text">
           <string>STAFF LOGIN</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #2c3e50; padding: 10px;</string>
          </property>
          <property name="text">
           <string>ADMIN PANEL</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="styleSheet">
           <string notr="true">background-color: #27ae60; color: white; padding: 10px; font-weight: bold;</string>
          </property>
          <property name="title">
           <string>Add New Medicine</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="qtyInput">
             <property name="placeholderText">
              <string>Quantity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="priceInput">
             <property name="placeholderText">
              <string>Price</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="nameInput">
             <property name="placeholderText">
              <string>Medicine Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="companyInput">
             <property name="placeholderText">
              <string>Company</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="idInput">
             <property name="placeholderText">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addMedicineBtn">
             <property name="text">
              <string>Add Medicine</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="adminLogoutBtn">
             <property name="styleSheet">
              <string notr="true">background-color: #95a5a6; color: white; padding: 10px;</string>
             </property>
             <property name="text">
              <string>Logout</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteBtn">
             <property name="styleSheet">
              <string notr="true">background-color: #e74c3c; color: white; padding: 10px;</string>
             </property>
             <property name="text">
              <string>Delete Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="updateBtn">
             <property name="styleSheet">
              <string notr="true">background-color: #f39c12; color: white; padding: 10px;</string>
             </property>
             <property name="text">
              <string>Update Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="searchAdminInput">
             <property name="styleSheet">
              <string notr="true">background-color: #e74c3c; color: white; padding: 10px;</string>
             </property>
             <property name="text">
              <string>Admin Search</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="searchStaffInput">
             <property name="styleSheet">
              <string notr="true">background-color: #f39c12; color: white; padding: 10px;</string>
             </property>
             <property name="text">
              <string>Staff Search</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="refreshAdminBtn">
             <property name="styleSheet">
              <string notr="true">background-color: #3498db; color: white; padding: 10px;</string>
             </property>
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="adminTable">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>300</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Company</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>New Column</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>742</width>
          <height>92</height>
         </rect>
        </property>
        <property name="title">
         <string>Sell Medicine</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="sellNameInput">
           <property name="placeholderText">
            <string>Medicine Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>STAFF PANEL - SELL MEDICINE</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="sellQtyInput">
           <property name="placeholderText">
            <string>Quantity</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sellBtn">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: #27ae60; color: white; padding: 12px; font-size: 16px; font-weight: bold;</string>
           </property>
           <property name="text">
            <string>SELL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextEdit" name="billDisplay">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>109</y>
          <width>744</width>
          <height>211</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"> background-color: #ecf0f1;
  padding: 15px;
  border: 2px solid #34495e;
  border-radius: 5px;
  font-family: monospace;
  font-size: 14px;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QTableWidget" name="staffTable">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>601</width>
          <height>250</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Company</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Price</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Stock</string>
         </property>
        </column>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>340</y>
          <width>131</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string>Actions</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QPushButton" name="staffLogoutBtn">
           <property name="styleSheet">
            <string notr="true">background-color: #95a5a6; color: white; padding: 10px;</string>
           </property>
           <property name="text">
            <string>Logout</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QPushButton" name="refreshStaffBtn">
           <property name="styleSheet">
            <string notr="true">background-color: #3498db; color: white; padding: 10px;</string>
           </property>
           <property name="text">
            <string>Refresh Stock</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
